### PR TITLE
Update and Switch to github actions

### PR DESCRIPTION
**Pull request by carkod**
## Done

Complete set of workflows in Github actions

## QA

Github actions checks should to turn green:
lint-js
lint-python
lint-scss
run-dotrun
run-image
test-js
test-python

### CODE REVIEW

**Comment by tbille** [comment]
@carkod why not moving all the jobs in the same PR?

**Comment by carkod** [comment]
@tbille done. But I'm still getting that strange error. The container runs correctly and outputs the usual server start output, but when it `curl` it returns a 500

**Comment by nottrobin** [comment]
Presumably you want to remove the circleci config file in this PR? Also it needs syncing with master.

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+          sudo chown root:root /
+
+      - name: Install dependencies
+        run: dotrun install
+
+      - name: Build assets
+        run: dotrun build
+
+      - name: Test site
+        run: dotrun & curl --head --fail --retry-delay 1 --retry 30 --retry-connrefused http://localhost:8012
```
Comment (nottrobin) [inline .github/workflows/pr.yaml:38]: Again, please put these commands on 2 separate lines

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          bash <(curl -s https://codecov.io/bash) -cF python
+    
+  lint-js:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: yarn
+      - name: Lint js
```
Comment (nottrobin) [inline .github/workflows/pr.yaml:99]: We normally separate steps with a blank line

File: .github/workflows/pr.yaml [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+        run: yarn lint-js
+
+  test-js:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: yarn
+      - name: Test js
```
Comment (nottrobin) [inline .github/workflows/pr.yaml:109]: Could we also add blank lines between the actions here?

File: Dockerfile [hunk 4]
```diff
@@ -21,6 +21,12 @@ FROM yarn-dependencies AS build-css
 ADD . .
 RUN yarn run build-css
 
+# Build stage: Run "yarn run build-css"
```
Comment (nottrobin) [inline Dockerfile:24]: This comment it wrong

**Pull request merged**